pkg/sanity/sanity.go

```diff
@@ -58,27 +58,44 @@ type Config struct {
 // gets initialized before each test block runs.
 type SanityContext struct {
 	Config  *Config
-	Conn    *grpc.ClientConn
 	Secrets *CSISecrets
 }
 
+var (
+	conn    *grpc.ClientConn
+	address string
+)
+
 // Test will test the CSI driver at the specified address by
 // setting up a Ginkgo suite and running it.
 func Test(t *testing.T, reqConfig *Config) {
 	sc := &SanityContext{
 		Config: reqConfig,
 	}
 
+	address = sc.Config.Address
 	registerTestsInGinkgo(sc)
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "CSI Driver Test Suite")
 }
 
+var _ = BeforeSuite(func() {
```

Comment:
I think @pohly 's change required these not to be set. Please check with him over slack.

Reply:
Yes, now it makes sense why this was removed. Embedding this test in an existing ginkgo setup with existing global `BeforeSuite` and `AfterSuite` fails to run. Maybe it's better to go with connection retry approach. The examples in ginkgo docs create multiple DB connections for each test case to avoid test pollution.

I don't have much experience with ginkgo. @pohly what do you think? Maybe there's some other better way to fix this?